code/modules/cargo/goodies.dm

```diff
@@ -4,6 +4,12 @@
 	group = "Goodies"
 	goody = TRUE
 
+/datum/supply_pack/goody/clear_pda
+	name = "Mint Condition Nanotrasen Clear PDA"
+	desc = "Mint condition, freshly repackaged! A valuable collector's item normally valued at over 2.5 million credits, now available for a steal!"
```

Comment:
it should come in a seamless preservative plastic box and wrap you need a knife to cut through (realism)